impala/tests/test_data_types.py

```diff
@@ -61,41 +61,40 @@ def date_table(cur):
     finally:
         cur.execute("DROP TABLE {0}".format(table_name))
 
-def setup_test_date_basic(cur, date_table):
+def setup_test_date(cur, date_table):
     """Insert and read back a couple of data values in a wide range."""
     cur.execute('select d from {0} order by d'.format(date_table))
     results = cur.fetchall()
     assert results == [(datetime.date(1, 1, 1),), (datetime.date(1999, 9, 9),)]
 
 @pytest.mark.connect
 def test_date_basic(cur, date_table):
-    setup_test_date_basic(cur, date_table)
+    setup_test_date(cur, date_table)
 
 @pytest.mark.connect
-def test_date_basic_noconv(cur_noconv, date_table):
-    setup_test_date_basic(cur_noconv, date_table)
+def test_date_no_string_conv(cur_no_string_conv, date_table):
+    setup_test_date(cur_no_string_conv, date_table)
 
 @fixture(scope='module')
 def timestamp_table(cur):
     table_name = 'tmp_timestamp_table'
     ddl = """CREATE TABLE {0} (ts timestamp)""".format(table_name)
     cur.execute(ddl)
+    cur.execute('''insert into {0}
+                   values (cast("1400-01-01 00:00:00" as timestamp)),
+                          (cast("2014-06-23 13:30:51" as timestamp)),
+                          (cast("2014-06-23 13:30:51.123" as timestamp)),
+                          (cast("2014-06-23 13:30:51.123456" as timestamp)),
+                          (cast("2014-06-23 13:30:51.123456789" as timestamp)),
+                          (cast("9999-12-31 23:59:59" as timestamp))'''.format(table_name))
     try:
         yield table_name
     finally:
         cur.execute("DROP TABLE {0}".format(table_name))
 
 
-@pytest.mark.connect
-def test_timestamp_basic(cur, timestamp_table):
+def setup_test_timestamp(cur, timestamp_table):
```

Comment:
Same as for dates, at this point this function is not about setup.

Reply:
Renamed